### DISCUSSION TITLE
Bluetooth connection doesn't work when I create dfu-file by myself

### DISCUSSION THREAD

**Discussion by 5kat3R** [InfiniTimeOrg/InfiniTime#1702] [Q&A]
Hi, 

It's been apporx. one year since I last built update for my Pinetime and everything worked great.
Now when I try to build a dfu-file and send (OTA) it to my Pinetime, everything goes normally except after restart the watch doesn't have a Bluetooth connection.

Any thoughts that what could be the reason? 

I build the dfu by using these commands:
`cmake -DARM_NONE_EABI_TOOLCHAIN_PATH=/home/user/gcc-arm-none-eabi-10.3-2021.10/ -DNRF5_SDK_PATH=/home/user/nRF5_SDK_15.3.0_59ac345/ -DBUILD_DFU=1`

`make -j4`

and I don't get any errors which makes it even more strange.

**Comment by tigercoding56** [comment]
try reconnecting in a while true loop

**Reply by 5kat3R** [reply]
It is a GadgetBridge issue:
https://github.com/InfiniTimeOrg/InfiniTime/issues/1724
